libs/sdk-flutter/lib/native_toolkit.dart

```diff
@@ -4,12 +4,15 @@ import 'bridge_generated.dart';
 
 BreezSdkCore? _breezSDK;
 
+const _libName = "libbreez_sdk_core.so";
+
 BreezSdkCore getNativeToolkit() {
   if (_breezSDK == null) {
-    final DynamicLibrary lib = Platform.isAndroid
-      ? DynamicLibrary.open("libbreez_sdk_core.so")   // Load the dynamic library on Android
-      : DynamicLibrary.process();
+    final DynamicLibrary lib = Platform.isIOS
```

Comment:
Perhaps it is better to be explicit here? e.g `if Platform.isLinux`
This way we can add Macos and Windows later.

Reply:
Sure thing, makes sense.